prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      min-height: 100vh;
+    }
+    #card-wrapper {
```

Comment:
Try using `.card-wrapper` to apply styles to every element with the `class` attribute of `"card-wrapper"`.

Reply:
![image](https://user-images.githubusercontent.com/106715786/173544580-8d78fc30-28da-4a8e-8b94-1a56d2b26e1d.png)

Reply:
![image](https://user-images.githubusercontent.com/106715786/173545761-9eb4acfb-5358-4623-9c57-75f80fce4811.png)
Should display: flex; be under #card-deck or .card-wrapper?